chain/client/src/client_actor.rs

```diff
@@ -1668,7 +1689,16 @@ impl ClientActor {
                                 )
                             })
                             .collect();
-
+                    self.start_state_sync_actors(ctx, &shards_to_sync);
```

Comment:
Note to self: This doesn't start a thread every few milliseconds, but checks if an actor has already been started.

Reply:
yea, name doesnt help with that. renamed to `ensure_sync_actors_running()`